### PR TITLE
Style updates per issues #162 & #161

### PR DESCRIPTION
**Pull request by stacietaylorcima**
Here are a few small updates to the new Coyote style. I'm seeing that my branch is 5 behind master, 1 ahead. To fix this, do I need to pull and re-push? What's the best practice for this? I tried to pull the master, but now my changes to be committed are all of the ones @flipsasser has already committed, so that doesn't seem right. Let me know what I should do. Thanks!

### CODE REVIEW

**Comment by flipsasser** [comment]
Don't worry about the merge - we can merge this on top of master cleanly, so it's not a big deal

File: app/views/representations/_audits.slim [hunk 1]
```diff
@@ -26,10 +26,9 @@ table
         td= to_html(text.to_s)
         td= User.find_by(id: author_id)&.username
         td= content_type
-        td= language
-        td= status
-        td= ordinality
-        td= Metum.find_by(id: metum_id)&.title
+        td: span.tag.tag--outline= representation.language
+        td= representation_status_tag(representation)
+        td= metum_tag(representation.metum, hint: false)
```
Comment (flipsasser) [inline app/views/representations/_audits.slim]: Did you mean to remove ordinality from here? If so, you should remove it from the header `th`'s as well

Comment (stacietaylorcima) [reply app/views/representations/_audits.slim]: Ahhh... I see what happened here. I didn't have ordinality on my branch locally and accidentally removed it during the merge conflict resolution here in Github. When I saw this, I went to put it back in the code locally, but it wasn't there, obviously. So, what is the best way to get it back in the code now? @flipsasser

Comment (stacietaylorcima) [reply app/views/representations/_audits.slim]: Hmmm... Well, when I access _audits in Github under this branch, I do see the th for Order, but I don't see it locally. Maybe I am just really confused about what's being displayed in Github. I could add the th and td for ordinality in locally and push if that seems right? @flipsasser #totalnewb

**Pull request merged**